.github/workflows/continuous-integration.yml

```
@@ -9,7 +9,7 @@ on:
 jobs:
   check-dist:
     name: Check dist/ directory
-    uses: actions/reusable-workflows/.github/workflows/check-dist.yml@a8533f184b279cfc1b2dd6a96ed2f097ccf81189
+    uses: actions/reusable-workflows/.github/workflows/check-dist.yml@97a652b80035363df47baee5031ec8670b8878ac # 4.0.0
```

Was moving to merge this and noticed this change, I don't see this commit SHA on `actions/reusable-workflows`, did you perhaps mean this?

```suggestion
    uses: actions/reusable-workflows/.github/workflows/check-dist.yml@967035ce963867fb956a309c9b67512314bc7c1f
```

I'm not sure why the GH workflow fail didn't show in the PR 🙃 
https://github.com/hashicorp/setup-terraform/actions/runs/6477487204/workflow

---

not sure how I got it, let me revert these to latest main.